ControlRoom/Controllers/SimCtl.swift

```diff
@@ -30,6 +31,7 @@ enum SimCtl {
     }
 
     private static func execute(_ arguments: [String]) -> PassthroughSubject<Data, SimCtl.Error> {
+        print("SIMCTL-CMD:", arguments.joined(separator: " "))
```

Comment:
will remove these before merge, just want to make sure commands are coming out correctly